### PR TITLE
Fix: RTL layout of read more save buttons

### PR DESCRIPTION
**Pull request by niedzielski**
Fix right-to-left layout of read more save buttons by separating out button and label. This will also allow for platform specific styling.

Before and after (left and right):

localhost-3000-footertransformer html-platform android layout_direction content-ltr theme default page_protection unprotected
localhost-3000-footertransformer html-platform android layout_direction content-ltr theme default page_protection unprotected

localhost-3000-footertransformer html-platform android layout_direction content-rtl theme default page_protection unprotected
localhost-3000-footertransformer html-platform android layout_direction content-rtl theme default page_protection unprotected

localhost-3000-footertransformer html-platform ios layout_direction content-ltr theme default page_protection unprotected
localhost-3000-footertransformer html-platform ios layout_direction content-ltr theme default page_protection unprotected

localhost-3000-footertransformer html-platform ios layout_direction content-rtl theme default page_protection unprotected
localhost-3000-footertransformer html-platform ios layout_direction content-rtl theme default page_protection unprotected

### CODE REVIEW

**Comment by montehurd** [comment]
Oh cool I'll review this tomorrow. (the save buttons are working RTL on iOS already though iirc... strange)

**Comment by montehurd** [comment]
Confirmed that the RTL layout of save buttons only seems to be manifesting on the demo:

screen shot 2017-08-21 at 11 37 58 am

Any ideas as to why?

File: changelog.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ### v4.5.7
 - Fix: catch failed read more request Errors
+- Fix: read more footer save button RTL layout
 - Update: footer wordmark
+- Update: improve read more save page button presentation abilities
```
Comment (montehurd) [inline changelog.md]: I updated the branch this is to be merged into to master and the changelog was updated based on merging your other PR and natalia's, so you may want to update the changelog.

**Comment by montehurd** [comment]
Looks like the save button issue was demo-specific. If I add this line:
`html.dir = (directionClass === 'content-rtl') ? 'rtl' : 'ltr'`

To the `ControlPanel.html`'s `onLayoutDirectionChange` it works as is.

fix mov

**Comment by montehurd** [comment]
That's not to say some of your changes may not be needed to meet Android requirements, but this may help reduce the amount of change needed (if confirmed that on Android you're setting `html.dir` which I thought you are). Also may help determine why this PR makes the save buttons disappear on iOS?

**Comment by niedzielski** [comment]
@montehurd, I've tried to repro on Chromium, Chrome, Firefox, Opera, and Epiphany but no luck. I've reverted changing the save button from a div to an inline-block span and hope this fixes it. I've still left off `tabel-cell` display which I hope won't be necessary. Please give this another spin. If you can, please also test in Safari assuming the [demo polyfill PR](https://github.com/wikimedia/wikimedia-page-library/pull/76) works.

**Comment by montehurd** [comment]
@niedzielski I pulled the latest and did another trial integration on iOS - still no save buttons.

**Comment by montehurd** [comment]
It looks like the first line in `updateSaveButtonForTitle` never succeeds in getting a button:
     `var saveButton = document.querySelector('#' + SAVE_BUTTON_ID_PREFIX + title);`

**Comment by montehurd** [comment]
@niedzielski feel free to throw a pair-debugging calendar invite my way if that would help!

**Comment by montehurd** [comment]
Closing for now - added note to circle back and re-review later.

**Pull request closed without merging**